packages/common/src/ui/components/inputs/Autocomplete/Autocomplete.tsx

```diff
@@ -73,6 +73,9 @@ export function Autocomplete<T>({
       sx={{ width }}
     />
   );
+  const getOptionLabel = (option: { label?: string } & T): string => {
```

Comment:
this is just to fix a complaint when you navigate to the detail page of an invalid shipment, where you get this:

![Screen Shot 2022-01-12 at 9 47 07 AM](https://user-images.githubusercontent.com/9192912/149026976-d233c68f-c332-4d01-af99-501fde6c2c7e.png)

because the Autocomplete `getOptionLabel` func defaults to `option.label ?? option` 

maybe not needed? should fix at the caller not here?

Reply:
Well I think I've seen this several times so obviously keep making the same mistake! Probably a good idea to just do it here, thanks!